src/v/redpanda/application.cc

```diff
@@ -582,12 +582,8 @@ void application::initialize(
      * Disable the logger for protobuf; some interfaces don't allow a pluggable
      * error collector.
      */
-#if PROTOBUF_VERSION < 5027000
```

Comment:
I think there are a couple other version checks - could you verify?

Reply:
> I think there are a couple other version checks - could you verify?

Good call, done.